e2e/testsuite/testsuite.go

```diff
@@ -154,12 +156,18 @@ func (s *E2ETestSuite) ConfigureRelayer(ctx context.Context, chainA, chainB ibc.
 	pathName := s.generatePathName()
 
 	channelOptions := ibc.DefaultChannelOpts()
-	// For now, set the version to the latest transfer module version
-	// DefaultChannelOpts uses V1 at the moment
-	channelOptions.Version = transfertypes.V2
-
 	if channelOpts != nil {
 		channelOpts(&channelOptions)
+	} else {
+		chainAVersion := chainA.Config().Images[0].Version
+		chainBVersion := chainB.Config().Images[0].Version
+
+		// select the transfer version based on the chains' version
+		transferVersion := transfertypes.V1
+		if testvalues.ICS20v2FeatureReleases.IsSupported(chainAVersion) && testvalues.ICS20v2FeatureReleases.IsSupported(chainBVersion) {
+			transferVersion = transfertypes.V2
+		}
+		channelOptions.Version = transferVersion
```

Comment:
In case the channel options is nil, I still want to figure out what's the transfer version that both chains support instead of setting by default V2.